sdk/src/test/java/io/opentelemetry/sdk/trace/TracerSdkProviderTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.Test;
+
+public class TracerSdkProviderTest {
+
+  @Test
+  public void testDefault() {
```

Comment:
What about a test that OpenTelemetry.getTracer returns an instance of the TracerSdk?

Reply:
Could you please clarify what are you asking? 

`OpenTelemetryTest` should test it:

```java
  @Test
  public void testDefault() {
    assertThat(OpenTelemetry.getTracer()).isInstanceOf(TracerSdk.class);
    assertThat(OpenTelemetry.getMeter()).isInstanceOf(DefaultMeter.getInstance().getClass());
    assertThat(OpenTelemetry.getTagger()).isInstanceOf(DefaultTagger.getInstance().getClass());
  }
```

Reply:
@Test
  public void loadTraceSdk() {
    assertThat(OpenTelemetry.getTracer()).isInstanceOf(TracerSdk.class);
  }

Reply:
I see you said in the OpenTelemetryTest. That is also fine.